doc/JSON_INFO.md

```diff
@@ -3732,6 +3732,22 @@ Add a map special to the starting location, see JSON_FLAGS for the possible spec
 
 A list of mission ids that will be started and assigned to the player at the start of the game. Only missions with the ORIGIN_GAME_START origin are allowed. The last mission in the list will be the active mission, if multiple missions are assigned.
 
+## `custom_initial_date`
+(optional, object with optional members "hour", "day", "season" and "year")
+
+Allows customizing initial date. If not set - corresponding values from world options are used. Random value is used for each parameter that is not explicitly.
```

Comment:
```suggestion
Allows customizing initial date. If not set - corresponding values from world options are used. Random value is used for each parameter that is not explicitly set in world options.
```
I think there's something missing at the end of this sentence.  Is this suggestion correct?

Reply:
He means, "If there is a custom_initial_date entry, each sub-memeber that is not set will get a random value".